brevent/src/main/res/values/strings.xml

```diff
@@ -168,7 +168,7 @@
     <string name="brevent_optimize_audio">Optimize music player</string>
     <string name="brevent_optimize_audio_label">Won\'t be brevented while playing with notification, won\'t be force-stopped while pausing with notification</string>
     <string name="brevent_auto_aggressive">Auto aggressive</string>
-    <string name="brevent_auto_aggressive_label">If app is ran with notifications as service, set it aggressive</string>
+    <string name="brevent_auto_aggressive_label">If app is ran with notification service, set it to aggressive</string>
```

Comment:
It does not mean `notification service`.

Reply:
gotchu, will fix